.github/workflows/container-structure-test.yml

```
@@ -15,7 +15,7 @@ jobs:
       - name: Install Google Container Structure Test Framework
         run: curl -LO https://storage.googleapis.com/container-structure-test/latest/container-structure-test-linux-amd64 && chmod +x container-structure-test-linux-amd64 && sudo mv container-structure-test-linux-amd64 /usr/local/bin/container-structure-test
```

We could add docker cache to have a shorter build, but not sure it's a good idea because we are in test process.

```suggestion
        run: curl -LO https://storage.googleapis.com/container-structure-test/latest/container-structure-test-linux-amd64 && chmod +x container-structure-test-linux-amd64 && sudo mv container-structure-test-linux-amd64 /usr/local/bin/container-structure-test
      - uses: satackey/action-docker-layer-caching@v0.0.10
        continue-on-error: true
```

---

I have to admit I have no idea :/

---

In theory, that changes nothing as that bypass build step that didn't change, but it's tricky.